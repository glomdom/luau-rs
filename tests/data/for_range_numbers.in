fn main() {
    for x in 1..10 {}
}