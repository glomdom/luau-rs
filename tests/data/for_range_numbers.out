Block(
    Block {
        statements: [
            Function(
                Function {
                    name: "main",
                    params: [],
                    ret_type: None,
                    body: Block(
                        Block {
                            statements: [
                                ForIn(
                                    ForIn {
                                        vars: [
                                            "x",
                                        ],
                                        iter: Range(
                                            Range {
                                                start: Some(
                                                    Value(
                                                        Value {
                                                            value: "1",
                                                        },
                                                    ),
                                                ),
                                                end: Some(
                                                    Value(
                                                        Value {
                                                            value: "10",
                                                        },
                                                    ),
                                                ),
                                            },
                                        ),
                                        body: Block(
                                            Block {
                                                statements: [],
                                            },
                                        ),
                                    },
                                ),
                            ],
                        },
                    ),
                },
            ),
        ],
    },
)